mender/values.yaml

```diff
@@ -95,8 +95,8 @@ device_auth:
       memory: 128M
   affinity: {}
   image:
-    registry: docker.io
-    repository: mendersoftware/deviceauth
+    registry: ""
+    repository: ""
```

Comment:
Why are you setting these variables to empty string?

Reply:
All enterprise services have empty strings in `registry` and `repository`. I fixed if statement in deviceauth deplyoment so proper values are chosen both when OS and enterprise template is prepared. Thanks!